### PR TITLE
Update test files to showcase block level Razor statements in attributes.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- Block level Razor statements in attributes used to not work. Added a few use cases to ensure that they're properly tested.

aspnet/Razor#594

### CODE REVIEW

**Comment by dougbu** [comment]
@NTaylorMullen think you mean aspnet/Razor#594 and the aspnet/Razor#595 PR fixing that.

**Comment by dougbu** [comment]
:shipit:

File: test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml [hunk 1]
```diff
@@ -33,6 +33,7 @@
         <menuitem icon=~/Person />
     </menu>
     <object data="~/Person" archive="~/Person/@Model.Name" data="~/Person" archive="~/Person/@Model.Name" />
+    <object archive="@if(Model.Name != null) { <text>~/Person/@Model.Name</text> } else { <text>~/Person</text> } " />
```
Comment (pranavkm) [inline test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml]: Do we have coverage for `await` inside of one of these things?

Comment (NTaylorMullen) [inline test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml]: Not at runtime. At compile time yes. Worth adding?

Comment (dougbu) [inline test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml]: :+1: to @pranavkm's suggestion though the compiler is more likely to complain about an `async` lambda _without_ an `await`. Nice that it doesn't.

Comment (NTaylorMullen) [inline test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml]: Will add. Only reason why it doesn't complain about the lacking `@await` is due to the supression above `ExecuteAsync`.

Comment (dougbu) [inline test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml]: Still nice. But why would someone `@await` (emphasis on the `@`) _inside_ a code block? :smiling_imp:

**Comment by NTaylorMullen** [comment]
:up::date:

File: test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml [hunk 2]
```diff
@@ -33,6 +40,7 @@
         <menuitem icon=~/Person />
     </menu>
     <object data="~/Person" archive="~/Person/@Model.Name" data="~/Person" archive="~/Person/@Model.Name" />
+    <object archive="@if(Model.Name != null) { <text>~/Person/@Model.Name</text> } else { await DoSomething(); <text>~/Person</text>  } " />
```
Comment (pranavkm) [inline test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml]: :clap:

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**